### DISCUSSION TITLE
Adding gatsby-plugin-s3 although I'm not the maintainer

### DISCUSSION THREAD

**Discussion by davidrhoderick** [gatsby-uc/plugins#377] [Q&A]
If you look up how to to deploy a Gatsby project to an AWS S3 bucket, you'll get [this page](https://www.gatsbyjs.com/docs/how-to/previews-deploys-hosting/deploying-to-s3-cloudfront/), which points you in the direction of `gatsby-plugin-s3`.  However, it looks as if this plugin is no longer maintained as I've pointed out [here](https://github.com/jariz/gatsby-plugin-s3/issues/422).  Based on the discussion I started in the [gatsby](https://github.com/gatsbyjs/gatsby/discussions/37695) repo, I was suggested to add it here, and I think it would make a lot of sense to if it's part of the official documentation.

However, I am not the maintainer of this plugin.  I only just tried today to get involved in contributing without success and there haven't been any responses to issues in that repo in at least 6 months (and that response was to say that an issue was duplicated).

How could we revive this plugin and bring it into `gatsby-uc`?  Should we fork it here?  How would we go about the npm package?  Should we rename it?

**Comment by moonmeister** [comment]
Hey @davidrhoderick! Yeah, I would love to see that happen. @jariz opened an [issue](https://github.com/gatsby-uc/plugins/issues/24) a while back to do this but went silent. Ideally we'd get the repo and npm package from them but if not we can discuss options. 

I'll try reaching out again when I get sometime and see if I can get their attention to get this done. Thanks for bringing this up!

**Comment by moonmeister** [comment]
I'm going to try one last hail mary to get in touch with @jariz and if it fails we can move forward with forking and publishing under a new name.

**Reply by moonmeister** [reply]
hmm...this account name appears to have changed...need to figure some stuff out.

**Reply by YoshiWalsh** [reply]
I am the current maintainer. As Jari mentioned, my time is limited, so if someone's willing to take ownership of the project that suits me well.

I'd still be interested in helping to maintain the project if that's possible.

**Reply by moonmeister** [reply]
> I am the current maintainer. As Jari mentioned, my time is limited, so if someone's willing to take ownership of the project that suits me well.
> 
> I'd still be interested in helping to maintain the project if that's possible.

Our goal here is to enable anyone interested to help contribute and maintain. I personally don't have time but as I admin the GUC, i can help get it into this repo if that is what we all want. You and whomever else would continue with maintenance...though hopefully there are fewer bottlenecks to new maintainers. #24 can be picked up and finished if you'd like to move forward. No rush, let me know if you have any questions.

**Comment by jariz** [comment]
Hi @moonmeister, I've transferred the project over to @joshuawalsh, a long time collaborator on the project, and personally do not maintain it anymore.  
I'm sure he's probably interested in talking with you, as he also made clear to me that he does not have a whole lot of time for it.  
Hope you guys can figure something out to give the project a more stable future.

**Reply by jariz** [reply]
Also, if this @ does not end up summoning him, try reaching out to him on Reddit, that's generally how we conversed.   
His username is on his personal website. Good luck and thanks!

**Reply by moonmeister** [reply]
Thanks @jariz for the update!
